selfdrive/car/body/values.py

```diff
@@ -34,6 +34,7 @@ class CAR:
       bus=0,
     ),
   ],
+  ecus={(0x720, None): Ecu.engine, (0x721, None): Ecu.debug},
 )
 
 FW_VERSIONS = {
```

Comment:
```suggestion
FW_VERSIONS = {
  CAR.BODY: {
    Ecu.engine: [
```

Reply:
Hmm, maybe can't do this so easily. On Honda the same address is used for the fwdRadar and fwdCamera on a platform-by-platform basis. (0x18dab0f1)